index.js

```diff
@@ -0,0 +1,97 @@
+
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import ReportingRoute from './server/routes/reporting.js';
+
+export default function(kibana) {
+  return new kibana.Plugin({
+    name: 'reporting',
+    uiExports: {
+      app: {
+        title: 'Reporting',
+        description: 'Reporting Module',
+        main: 'plugins/reporting/app',
+      },
+    },
+    require:  ['kibana', 'elasticsearch'],
+
+    config(Joi) {
+      return Joi.object({
+        enabled: Joi.boolean().default(true),
+        required_backend_role: Joi.string().allow('').default(''),
+      }).default();
+    },
+
+    init(server, options) {
+      const esDriver                = server.plugins.elasticsearch;
+      const esServer                = server.plugins;
+      const GenerateReportService   = require('./server/services/GenerateService');
```

Comment:
any reason use require instead import to load module?

why load module in init()?